python/cugraph/CMakeLists.txt

```diff
@@ -36,6 +36,22 @@ option(FIND_CUGRAPH_CPP "Search for existing CUGRAPH C++ installations before de
 
 # If the user requested it,  we attempt to find CUGRAPH.
 if(FIND_CUGRAPH_CPP)
+
+  include(rapids-cpm)
+  rapids_cpm_init()
+  include(rapids-cuda)
+  rapids_cuda_init_architectures(cugraph-python)
+  enable_language(CUDA)
+
+  set(CUGRAPH_VERSION_MAJOR "${cugraph-python_VERSION_MAJOR}")
+  set(CUGRAPH_VERSION_MINOR "${cugraph-python_VERSION_MINOR}")
+
+
+  # We need to call get_raft due to cuML asking for raft::nn and
+  # raft::distance targets
+  # see issue https://github.com/rapidsai/cuml/issues/4843
+  include(../../cpp/cmake/thirdparty/get_raft.cmake)
```

Comment:
@cjnolet good find. The underlying issue here will be fixed by https://github.com/rapidsai/rapids-cmake/pull/154.